src/model/guild/member.rs

```diff
@@ -92,7 +97,7 @@ impl Member {
                 self.roles.push(role_id);
 
                 Ok(())
-            },
+            }
```

Comment:
These changes kept getting applied every time I ran RustFmt. Let me know if you want me to undo them.

Reply:
These commas are meant to be there, as [defined by our rustfmt configuration](https://github.com/serenity-rs/serenity/blob/current/rustfmt.toml#L11). Which version of rustfmt are you using? We only support nightly, because we use options not yet supported in stable rustfmt.

Reply:
I see. I was not using nightly -- I was letting an editor plugin do the formatting for me, which was using the default rustfmt channel. Running `cargo +nightly fmt` resolved the commas and other formatting discrepancies. Thank you.